packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -205,9 +231,11 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
       onMouseEnter={() => setItemHovered(true)}
       onMouseLeave={() => setItemHovered(false)}
       onClick={() => {
-        setItemHovered(true);
-        setMenuHidden(false);
-        onClick?.(props);
+        if (!participantStateString) {
```

Comment:
this scares me, as a developer its not clear that if I set the state suddenly my menu item button is gone

Reply:
Noted!
Have this as a todo item.